### PR TITLE
sway: update to 1.1.1

### PR DESCRIPTION
**Pull request by julio641742**
+ wlroots: update to 0.6.0

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -3319,7 +3319,7 @@ libcodecore.so.0 libio.elementary.code-3.0_1
 libio.elementary.music-core.so.0 libio.elementary.music-5.0_1
 libpantheon-files-core.so.4 libio.elementary.files-4.1.4_1
 libpantheon-files-widgets.so.4 libio.elementary.files-4.1.4_1
-libwlroots.so.2 wlroots-0.5.0_1
+libwlroots.so.3 wlroots-0.6.0_1
```
Comment (xtraeme) [inline common/shlibs]: wlroots soname has been bumped, but there are revdeps that need to be rebuilt...

try `xbps-query -RX wlroots`

File: srcpkgs/wlroots/template [hunk 2]
```diff
@@ -1,8 +1,7 @@
 # Template file for 'wlroots'
 pkgname=wlroots
-reverts="0.6.0_1"
-version=0.5.0
-revision=2
+version=0.6.0
+revision=1
```
Comment (jnbr) [inline srcpkgs/wlroots/template]: We already had `wlroots-0.6.0_1` by accident. You have to set revision to 2 to avoid confusion.

Comment (julio641742) [reply srcpkgs/wlroots/template]: Done

File: common/shlibs (hunk 1 above)
Comment (julio641742) [reply common/shlibs]: wayfire is broken, they have not caught up with wlroots 0.6.0

**Comment by tjg1** [comment]
wayfire is already fixed in their git master, I poked them to tag a release WayfireWM/wayfire#232

File: srcpkgs/sway/template [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-version=1.0
+version=1.1.1
 revision=1
 build_style=meson
 conf_files="/etc/sway/config"
 hostmakedepends="pkg-config wayland-devel scdoc git"
 makedepends="wlroots-devel wayland-devel wayland-protocols pcre-devel
  json-c-devel pango-devel cairo-devel gdk-pixbuf-devel"
-depends="xorg-server-xwayland"
+depends="swaybg xorg-server-xwayland"
```
Comment (CameronNemo) [inline srcpkgs/sway/template]: Technically it functions without swaybg.

Comment (CameronNemo) [reply srcpkgs/sway/template]: Note we do not have swayidle or swaylock as deps. Could be useful create a sway meta package.

Comment (Ypnose) [reply srcpkgs/sway/template]: From @ddevault in 1.1 desc:

> swaybg is a required dependency of sway. Packagers are encouraged to configure it as such when updating sway for their distribution.

Comment (CameronNemo) [reply srcpkgs/sway/template]: Ah! Thanks for the clarification.

File: common/shlibs (hunk 1 above)
Comment (julio641742) [reply common/shlibs]: wayfire 0.2 is up to date with wlroots

**Comment by julio641742** [comment]
this pull request is ready for another review

File: srcpkgs/wlroots/template [hunk 4]
```diff
@@ -1,7 +1,6 @@
 # Template file for 'wlroots'
 pkgname=wlroots
-reverts="0.6.0_1"
-version=0.5.0
+version=0.6.0
 revision=2
```
Comment (xtraeme) [inline srcpkgs/wlroots/template]: revision=1

Comment (CameronNemo) [reply srcpkgs/wlroots/template]: but we already had a revision 1? wlroots was bumped then reverted.

Comment (CameronNemo) [reply srcpkgs/wlroots/template]: see @jnbr comment above

Comment (xtraeme) [reply srcpkgs/wlroots/template]: My fault, didn't look at `reverts` :-)

**Pull request merged**